### PR TITLE
add publish artifact task to pipeline for cpp console

### PR DESCRIPTION
**Pull request by splumber**
## Purpose
* Add publish artifact task for cpp console

## Does this introduce a breaking change?
[ ] Yes
[x] No

## Pull Request Type
What kind of change does this Pull Request introduce?

[ ] Bugfix
[x] Feature
[ ] Code style update (formatting, local variables)
[ ] Refactoring (no functional changes, no api changes)
[ ] Documentation content changes
[ ] Other... Please describe:

### CODE REVIEW

File: azure-pipelines.yml [hunk 1]
```diff
@@ -53,7 +53,7 @@ variables:
   sample3Name: VoiceAssistantCPPClient
   sample3Folder: 'clients\cpp-console'
   sample3Solution: '$(sample3Folder)\src\windows\cppSample.sln'
-  sample3PublishedArtifactPath: '$(sample3Folder)\out'
+  sample3PublishedArtifactPath: '$(sample3Folder)\$(sample3Name)\bin\$(buildPlatformNative)\$(buildConfiguration)'
   sample3PublishedArtifactName: 'CppSample-$(Build.BuildNumber)'
```
Comment (dargilco) [inline azure-pipelines.yml:58]: sample3PublishedArtifactName: '$(sample3Name)-$(Build.BuildNumber)'

Comment (dargilco) [reply azure-pipelines.yml:58]: Please do the above before merge. Thanks!

Comment (splumber) [reply azure-pipelines.yml:58]: Done

**Comment by dargilco** [comment]
Looks good. Merge.

**Pull request merged**